doc/changes/changes-4.10.xml

```diff
@@ -0,0 +1,312 @@
+<Chapter Label="ChangesGAP49toGAP410"> 
+<Heading>Changes between &GAP; 4.9 and &GAP; 4.10</Heading>
+
+This chapter contains an overview of the most important changes
+introduced in &GAP; 4.10.0 release (the 1st public release of &GAP; 4.10).
+Later it will also contain information about subsequent update 
+releases for &GAP; 4.10.
+<P/>
+
+These changes are also listed on the Wiki page
+<URL>https://github.com/gap-system/GAP/wiki/gap-4.10-release-notes</URL>.
+
+<Section Label="GAP4.10.1">
+<Heading>&GAP; 4.10.0 (November 2018)</Heading>
+
+<Subsection> 
+<Heading>New features and major changes</Heading>
+
+<List>
+
+<!-- 
+ -->
+<Mark>Reduce impact of immediate methods</Mark><Item>
+
+GAP allows declaring so-called "immediate methods". The idea is that these
+are very simple and fast methods which are immediately called if
+information about an object becomes known, in order to perform some quick
+deduction. For example, if the size of a group is set, there might be
+immediate methods which update the filters <C>IsFinite</C> and <C>IsTrivial</C> of the
+group suitably.
+<P/>
+
+While this can be very elegant and useful in interactive GAP sessions, the
+overhead for running these immediate methods and applying their results can
+become a major factor in the runtime of complex computations that create
+thousands or millions of objects.
+<P/>
+
+To address this, various steps were taken:
+<List>
+<Item>some immediate methods were turned into regular methods;</Item>
+<Item>a special handlers for <C>SetSize</C> was created that performs deductions which previously were taken care of by immediate methods;</Item>
```

Comment:
perform (in singular)

Reply:
Huh? That's not a plural s. Anyway, I think "performs" sounds odd, better would be "... that deduced properties"

Reply:
a special handlers for <C>SetSize</C> was created that performs deductions

should be ether 

    a special handler for <C>SetSize</C> was created that performs deductions 

or

    special handlers for <C>SetSize</C> were created that perform deductions